enzy_htp/structure/structure_io/pdb_io.py

```diff
@@ -172,6 +172,18 @@ def get_file_str(cls, stru: Residue, if_renumber: bool = True, if_fix_atomname:
         result_str = cls._write_pdb_residue(stru)
         result_str += f"TER{os.linesep}END{os.linesep}"
         return result_str
+    
+    @classmethod
+    @dispatch
+    def get_file_str(cls, stru: ModifiedResidue, if_renumber: bool = True, if_fix_atomname: bool = True) -> str:  # pylint: disable=function-redefined
+        """
+        dispatch for supporting get pdb file str with Residue only
+        """
+        if if_renumber:
+            stru.parent.parent.renumber_atoms()
+        result_str = cls._write_pdb_residue(stru)
+        result_str += f"TER{os.linesep}END{os.linesep}"
+        return result_str
 
```

Comment:
This seems not necessary since ModifiedResidue is a child class of Residue. Can you try if the test work w/o this function?

Reply:
That's what I thought too, but without it, the test throws a plum.resolver.NotFoundLookupError